### PR TITLE
Updates for migration script

### PR DESCRIPTION
**Pull request by angaither**
#### What's this PR do?
- allow `gif` file types
- add a param to the array to not force forwards back to phoenix

#### How should this be reviewed?
👀 

#### Any background context you want to provide?
NO

#### Relevant tickets
Fixes #the issue from phoenix

### CODE REVIEW

File: app/Services/AWS.php [hunk 1]
```diff
@@ -30,7 +30,7 @@ public function storeImage($file, $filename)
         $extension = $this->guessExtension($data);
 
         // Make sure we're only uploading valid image types
-        if (! in_array($extension, ['jpeg', 'png'])) {
+        if (! in_array($extension, ['jpeg', 'png', 'gif'])) {
             throw new UnprocessableEntityHttpException('Invalid file type. Upload a JPEG or PNG.');
```
Comment (katiecrane) [inline app/Services/AWS.php]: doesn't matter for functionality but should we add gif to this list too?

**Comment by sbsmith86** [comment]
Looks good

**Comment by angaither** [comment]
@katiecrane updated :framed_picture:

**Comment by katiecrane** [comment]
@angaither 🖊  looks good to me!

**Pull request merged**